scripts/capstone.py

```diff
@@ -141,7 +189,9 @@ def do_invoke(self, _: List[str], **kwargs: Any) -> None:
 
         insns = []
         opcodes_len = 0
-        for insn in cs_disassemble(location, length, skip=length * self.repeat_count, **kwargs):
+        for insn in cs_disassemble(
+            location, length, skip=length * self.repeat_count, **kwargs
+        ):
```

Comment:
I don't think we need (nor do I like) the hanging parens. Wastes lines imo.
```suggestion
        for insn in cs_disassemble(
            location, length, skip=length * self.repeat_count, **kwargs):
```